Code/Framework/AzToolsFramework/AzToolsFramework/ComponentMode/ComponentModeSwitcher.cpp

```diff
@@ -135,7 +138,7 @@ namespace AzToolsFramework::ComponentModeFramework
                 {
                     // if two or more entities are selected, ensure the only components
                     // that remain on the switcher are common to all entities
-                    if (selectedEntityIds.size() > 1 && !m_addedComponents.empty())
+                    if (selectedEntityIds.size() > 1)
```

Comment:
Does this `!m_addedComponents.empty()` check still need to go away?

Reply:
I removed this because in the case where an entity has no components on the switcher and another entity is selected it doesn't go through the block to find which components are common between them.